code/game/gamemodes/steal_items.dm

```diff
@@ -28,6 +28,11 @@
 			return 1
 	return 0
 
+//This proc is to be used for not granting objectives if a special requirement other than job is not met.
+
+/datum/theft_objective/proc/meets_objective_condition()
```

Comment:
```suggestion
/// This proc is to be used for denying objectives if there are other special requirements that need to be met
/datum/theft_objective/proc/meets_objective_condition()
```
I recommend renaming this proc to `check_objective_conditions()`

Reply:
Done